src/integrationTest/sam.test.ts

```diff
@@ -37,10 +38,10 @@ const scenarios: TestScenario[] = [
     {
         runtime: 'nodejs10.x',
         path: 'hello-world/app.js',
-        debugSessionType: 'node2',
+        debugSessionType: 'pwa-node',
         language: 'javascript',
     },
-    { runtime: 'nodejs12.x', path: 'hello-world/app.js', debugSessionType: 'node2', language: 'javascript' },
+    { runtime: 'nodejs12.x', path: 'hello-world/app.js', debugSessionType: 'pwa-node', language: 'javascript' },
```

Comment:
Would we be able to enable the c# test (line 49) with this change as well?

Reply:
we can try it. but it was disabled for reason unrelated to this PR.

Reply:
Right. The new debug implementation is an opportunity/expectation to onboard stable testing across the supported runtimes.

Reply:
Ok, thanks for checking